Swipeable.js

```diff
@@ -351,10 +351,10 @@ const styles = StyleSheet.create({
   },
   leftActions: {
     ...StyleSheet.absoluteFillObject,
-    flexDirection: 'row',
+    flexDirection: I18nManager.isRTL? 'row-reverse': 'row',
   },
   rightActions: {
     ...StyleSheet.absoluteFillObject,
-    flexDirection: 'row-reverse',
+    flexDirection: I18nManager.isRTL ? 'row' : 'row-reverse',
   },
-});
+});
```

Comment:
Line break at the end of file is missing :/

Reply:
I can't find it as not fixed. Could it be because I fixed it in my fork after sending the PR?

Reply:
OK I think I managed to sync the PR